xml-parser/src/main/java/org/uniprot/core/xml/uniref/AbstractMemberConverter.java

```diff
@@ -162,6 +170,9 @@ protected void updateMemberFromXml(
                     builder.uniref50Id(new UniRefEntryIdBuilder(property.getValue()).build());
                     break;
                 case PROPERTY_SOURCE_UNIPROT:
+                    if (accession == null) {
+                        accession = property.getValue();
+                    }
```

Comment:
Can there be more than one `UniProtKB accession`? If yes, then fine otherwise we don't need the if condition. I prefer Objects.isNull instead of dangling null.

Reply:
Yes, we can have many